elimu_yangu/sql/university_finder.sql

```diff
@@ -2,55 +2,50 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 9.5.10
--- Dumped by pg_dump version 9.5.10
-
--- Started on 2018-04-09 12:07:22 EAT
+-- Dumped from database version 9.5.13
+-- Dumped by pg_dump version 10.5
 
 SET statement_timeout = 0;
 SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
```

Comment:
Remove `SET idle_in_transaction_session_timeout = 0;` (not a valid configuration for PG `9.5`)